### ISSUE TITLE
Testing Skill error

### ISSUE THREAD

**Issue by SparJar** [babybuddy/babybuddy-alexa-skill#14]
Let me preface this with I am no developer.  But I have been getting a lot of use out of Baby Buddy and would like to link this skill up to my server.  

I have gone through all the steps and got the skill deployed, but when I test it in my browser Alexa just says "Sorry, I had trouble doing what you asked. Please try again."

Is there anything I am missing that I have to do to get this skill to work?  I did nothing other than follow the instructions given here. I can post any logs or anything else needed to help.

**Comment by cdubz** [comment]
Do you see any logs to post? I'm not familiar enough with the interface myself to give great guidance here but I do feel like I have run in to similar behavior in testing skills. And I assume you're using the latest version of the code on the `master` branch? I haven't actually tested a full deploy of it recently myself.

**Comment by SparJar** [comment]
Yes I am using the latest version of the master branch.  When I looked at it some more last night, in the Skills console on the "Code" tab it said I couldn't look because the code wasn't hosted in AWS blah blah. So I told it to convert to being hosted in AWS and then tested again and now the code run and I get a proper response in the test, but it's like it doesn't communicate with my BabyBuddy server properly as nothing happens there in my tests. I have the debug logs that get generated in a paste bin that I will put below.  I am a sysadmin by trade so I know how to read logs to a fair degree and what I don't see in them anywhere is where it is trying to contact my babybuddy server.  I am sure I am missing something simple, but my server is accessible from the internet and I did put that server address and api key in the build while I built the skill. I have poked my way around the Skill build properties and all as well and I do not see my server in there anywhere either.  So it may be that I just need to be told where to look for that to make sure it's actually trying to correct server.    

https://pastebin.com/zXQBPDSm
